### ISSUE TITLE
TinkersConstruct swords not effective on cowebs

### ISSUE THREAD

**Issue by Tomash667** [squeek502/WailaHarvestability#51]
Without WailaHarvestability broadswords can destroy cowebs very fast. But when this mod is installed it marks broadsword as not effective and it takes lots of time to destroy single coweb.

Take look at screenshots here:
https://github.com/SlimeKnights/TinkersConstruct/issues/2518

Using lastest version of TiC, Waila/Harvestability and forge.

**Comment by GirafiStudios** [comment]
Yeah that is indeed the case, very weird. I will look into this later :)

**Comment by GirafiStudios** [comment]
@squeek502 I have tracked it down to https://github.com/squeek502/WailaHarvestability/blob/1.7.10/java/squeek/wailaharvestability/setup/MissingHarvestInfo.java#L14. If you remove that code it works just like it should for both vanilla and Tinkers Construct swords. I'm not quite sure why you have done it like that though :)

**Comment by squeek502** [comment]
`Blocks.web.setHarvestLevel("sword", 0)` is done so that "Sword" shows up as the effective tool. Without that, nothing shows up at all (sword effectiveness is hardcoded in the web block code in a non-standard way IIRC). Would need to look at why the TiC tools are affected by this, though--maybe they set some other harvest level?

@Tomash667 which version of Minecraft are you using?

**Comment by GirafiStudios** [comment]
I assumed he was using latest 1.10.2 versions, which was the versions I was able to reproduce this with.

Ahh yeah you are right. Removing the code just fixes that the cob web aren't harvestable by TiCon swords, but it also removes the "Effective Tool" part.
I'm pretty sure the problem is that TiCon swords are not a sword in the vanilla/forge sense. As you can see in Tomash667s screenshots in the issue on TiCons issue tracker, Waila Harvestability does not think it is a sword.

Tinkers Construct does their cob web effectiveness code here: https://github.com/SlimeKnights/TinkersConstruct/blob/25a03943276b131e4dcbc74907c887491de343a8/src/main/java/slimeknights/tconstruct/library/tools/SwordCore.java

**Comment by squeek502** [comment]
Yeah, might be worth getting TiC to add `setHarvestLevel("sword", 0);` to SwordCore?

Also, just to make sure, this is a visual bug only, right? Waila Harvestability should only affect the client side so TiC swords should still be effective even if they don't visually appear to be.

**Comment by GirafiStudios** [comment]
No it is not a visual bug only. It actually makes you unable to break cob web with TiCon swords (Even though Waila Harvestability is client-side only, which is weird o.O)

**Comment by Tomash667** [comment]
@squeek502 It is as @GirafiStudios written. It's marked as not effective and can't break coweb.

Minecraft/forge version: 1.10.2-forge1.10.2-12.18.2.2111
TConstruct-1.10.2-2.5.6b
Mantle-1.10.2-1.1.1
Waila-1.7.0-B3_1.9.4
WailaHarvestability-mc1.10.2-1.1.9

**Comment by bonii-xx** [comment]
I added the sword compatibility, but you probably still should fix it on your side, since you're still changing the properties of vanilla cobwebs.

**Comment by squeek502** [comment]
@bonii-xx agreed, thanks for adding support on your end though.
